cmake/FindHiopSTRUMPACK.cmake

```diff
@@ -0,0 +1,42 @@
+
+#[[
+
+Exports target `STRUMPACK`
+
+Users may set the following variables:
+
+- HIOP_STRUMPACK_DIR
+- HIOP_SCALAPACK_DIR
+
+]]
+
+find_package(STRUMPACK CONFIG
+    PATHS ${STRUMPACK_DIR} ${HIOP_STRUMPACK_DIR}
+    REQUIRED)
+    
+find_library(SCALAPACK_LIBRARY
```

Comment:
I think the current version is just a temporary solution. Since Strumpack may be compiled with many other packages, we need to think about how to detect all its dependencies.
In my opinion, the ideal way is that the strumpack-config.cmake (generated by strumpack) should point out if ScaLapack is required and where to find it. HiOp can remove the dependency of scalapack if Strumpack can find it, as it can find and include metis, par-metis and other dependencies.